src/components/doc/Sidebar.js

```diff
@@ -117,6 +117,7 @@ const DocSidebar = () => {
           />
 
           <SidebarItem to="/docs/getting-started/adding-components/" text="Adding components" />
+          <SidebarItem to="/docs/getting-started/user-management/" text="User Management" />
```

Comment:
"Adding a user" is more consistent with "Adding components" in the sidebar item above.

```suggestion
          <SidebarItem to="/docs/getting-started/user-management/" text="Adding users" />
```